about.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Ethio Health Annual Exibition</title>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="about-style.css">
+</head>
+
+<body>
+	<header >
+		<nav class="header-nav">
+			<div class="container-top-nav">
+				<div class="header-top-nav">
+					<ul class="header-top-nav-desktop">
+						<li><a href="#" class="fa fa-facebook"></a></li>
+						<li><a href="#" class="fa fa-twitter"></a></li>
+						<li><a href="#">English</a></li>
+						<li><a href="#">My Page</a></li>
+						<li><a href="#">Logout</a></li>
+					</ul>
+				</div>
+			</div>
+			<div class="container-nav">
+				<div class="logo-nav">
+					<a href="./index.html">
+						<img src="./img/logo/ethio-health-logo.png" alt="logo Ethio health">
+					</a>
+				</div>
+				<ul class="header-nav-desktop">
+					<li><a href="./about.html">About</a></li>
+					<li><a href="#">Program</a></li>
+					<li><a href="#">Join</a></li>
+					<li><a href="#">Sponsor</a></li>
+					<li><a href="#">News</a></li>
+					<li><a href="#">Ticket</a></li>
+				</ul>
+			</div>
+		</nav>
+		<div class="container">
+			<div class="header-nav-phone">
+				<button type="button" class="menu-btn">
+					<img src="./img/icon/menu-icon.svg" alt="menu icon">
+				</button>
+				<nav class="header-nav-mobile hidden">
+					<button type="button" class="btn-close-nav">
+						<img src="./img/icon/close-icon.svg" alt="cancel icon">
+					</button>
+					<ul>
+						<li>
+							<a href="./index.html" >
+								Home
+							</a>
+						</li>
+						<li>
+							<a href="./about.html" >
+								About
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Program
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Join
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Sponsor
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								News
+							</a>
+						</li>
+						<li>
+							<a href="#" >
+								Ticket
+							</a>
+						</li>
+					</ul>
+				</nav>
+			</div>
+			<div class="headline">
+				<h3>
+					"Hello! Sharing world"
+				</h3>
+				<h1>
+					ETHIO HEALTH <br>EXHIBITION AND CONGRESS 2023
```

Comment:
- [ ] Please make sure that the color of the title in the about page matches the design 
![image](https://user-images.githubusercontent.com/74744787/218277363-3db1e535-dd5e-4767-8588-d27746ded38d.png)